shell.nix

```diff
@@ -16,6 +16,9 @@ mkShell {
     perl
     python3
     bazel
+    # For stack_install. TODO Remove ghc when move to Stack 2.
```

Comment:
What does Stack 2 improve in this respect?

Reply:
It generates the dependency graph without the spurious requirement that GHC be in the environment.